src/Form/ScheduleType.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+
+namespace App\Form;
+
+
+use App\Dto\ScheduleRequest;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\DateType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ScheduleType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $daysAvailables = [
+            ' -- Select -- ' => '',
+            '2019-06-08' => '2019-06-08',
```

Comment:
This is probably going to change I guess, but you should never ever hardcode a date, because at least you'd need internationalisation (i18n) at some point

Reply:
Yep, is date is as example, the idea is get in creation of event.
"The event will take place on 08 and 09 June"